src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-form container"
+      method="get"
```

Comment:
typically for method attribute we use uppercase 
```suggestion
      method="GET"
```